app/views/crime_applications/_overview.html.erb

```diff
@@ -46,7 +46,7 @@
       <%= t(:application_start_date, scope: 'crime_applications.labels') %>
     </dt>
     <dd class="govuk-summary-list__value">
-      <%= l(overview.application_start_date) %>
+      <%= l(overview.created_at) %>
```

Comment:
I'm not mad about calling this created_at. Would prefer application_started_at. (Thinking about re-submissions etc). But that should not block this PR.